src/cobald/daemon/config/python.py

```diff
@@ -1,5 +1,26 @@
-import include
+import pathlib
+import importlib.util
+import sys
+
+
+_loaded = -1
 
 
 def load_configuration(path):
-    return include.path(path)
+    """
+    Load a configuration from a module stored at ``path``
+
+    The ``path`` must end in a valid file extension for the appropriate module type,
+    such as ``.py`` or ``.pyc`` for a plaintext or bytecode python module.
+    """
+    global _loaded
+    current_index = _loaded = _loaded + 1
+    module_name = f"<cobald config {current_index}>"
+    spec = importlib.util.spec_from_file_location(module_name, path)
+    if spec is None:
+        extension = pathlib.Path(path).suffix
+        raise RuntimeError(f"Unrecognized file type {extension} for config {path}")
+    module = importlib.util.module_from_spec(spec)
```

Comment:
``importlib.util.module_from_spec`` ([link](https://docs.python.org/3/library/importlib.html#importlib.util.module_from_spec)) can in principle return `None`. Do you want to check for this or doesn't it hurt somewhere else? I think there might be errors when executing ``exec_module``, but I didn't dive into to code to check ;)

Reply:
I've reread the docs... several times. I *think* what they are saying is that the ``None`` occurs only internally.